src/backend/metal/src/command.rs

```diff
@@ -446,12 +453,17 @@ impl RawCommandBuffer<Backend> for CommandBuffer {
         }
     }
 
-    fn set_viewports(&mut self, vps: &[Viewport]) {
+    fn set_viewports<T>(&mut self, vps: T)
+    where
+        T: IntoIterator<Item=Viewport>,
+    {
+        let mut vps = vps.into_iter();
+        let vp = if let Some(first) = vps.next() {
+            first
+        } else {
+            return
```

Comment:
This is a change in behaviour, should this panic or exit silently?

Reply:
let's panic like it's 99th!